jinahub/indexers/storage/MongoDBStorage/tests/test_mongodb.py

```diff
@@ -0,0 +1,88 @@
+import os
+from pathlib import Path
+
+import pytest
+import numpy as np
+from jina import Document, DocumentArray, Flow, Executor
+from jina_commons.indexers.dump import import_vectors, import_metas
+
+from .. import doc_without_embedding
```

Comment:
This can then be replaced with 
```suggestion
from ..mongodb_storage import doc_without_embedding
```